doc/changelog/08-tools/11523-coqdep+refactor2.rst

```diff
@@ -0,0 +1,5 @@
+- **Changed:**
+  Internal options and behavior of `coqdep` have changed, be aware if you
+  are using these low-level features.
```

Comment:
Maybe could use a bit more detail (list of removed / changed options) to make it easy to tell when one is concerned.

Reply:
Updated, thanks.